Ed-Fi-Exchange-OSS/EdFi-Project-Buzz
# [FIF-344] Installs UI

Creates runtime-Config to allow users to configure a custom logo and URL to the API. Allows the EnvironmentService to load runtime configuration first, and default back to environment variables for local development.

NOTE: At this time the API is not installing correctly, set installApi to false in the configuration and run directly. This will be addressed in FIF-345.

## TO TEST LOCAL DEVELOPMENT
Run UI locally first in development to confirm that normal .env variables are populated.
Expect it to NOT populate the public/runtime-config.js file.

## TO TEST INSTALLER
Ensure that artifactRepo is set to C:\nugetRepo to get locally built packages.
Run the test.ps1 script first to confirm new packages are created and the temp directory packages are cleared.
Expect it to populate the public/runtime-config.js file with configuration.json variables and use those during execution.
